src/main/java/de/doubleslash/usb_led_matrix/view/AuthenticationView.java

```diff
@@ -185,6 +182,7 @@ public void setWebsiteCode() {
          this.deviceCode.set(deviceCode);
       } catch (IOException | InterruptedException e) {
          LOG.error("Could not retrieve device code: ", e);
+         Thread.currentThread().interrupt();
```

Comment:
guess we should only interrupt, if interrupt exception was thrown. not also in case of IOException. 
(same comment as was already resolved at another place)

Reply:
Is now only interrupted in the event of an interruption exception

Reply:
- extra ; present
- logging is different than in the other locations where u changed it. please do it consistently